### PR TITLE
Adding a new problem

### PR DESCRIPTION
**Pull request by AlerGames**
For assignment 3 of SF-1 ~~~ :3

### CODE REVIEW

File: sum-oddneven-index/description.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Write a function `sum_oddneven_list(a) -> int` that, given a non empty list of integers, return the sum of it’s even index values or of it’s odd index values depending on whether the length of the list is odd (sum the evens) or even (sum the odds).
+
+For example:
+- `sum([1,2,3,4,5,6,7,8,9,10]) → 2 + 4 + 6 + 8 + 10 = 30`
+- `sum(111, 1) → 1`
+- `sum(22,11,33) → 22 + 33 = 55`
```
Comment (samchouse) [inline sum-oddneven-index/description.md:6]:
```suggestion
- `sum_oddneven_list([1,2,3,4,5,6,7,8,9,10]) → 2 + 4 + 6 + 8 + 10 = 30`
- `sum_oddneven_list([111, 1]) → 1`
- `sum_oddneven_list([22,11,33]) → 22 + 33 = 55`
```

**Review by samchouse — CHANGES_REQUESTED** [review]
LGTM other than this :D

File: sum-oddneven-index/description.md (hunk 1 above)
Comment (AlerGames) [reply sum-oddneven-index/description.md:6]: very durr, thanks

**Review by samchouse — APPROVED** [review]

File: sum-oddneven-index/io.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+[
+	{
+		"input": [1,2,3,4,5,6,7],
```
Comment (emaj9) [inline sum-oddneven-index/io.json:3]: this needs double [[ ]]

**Pull request merged**